.github/workflows/performance-regression.yml

```
@@ -114,17 +114,17 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v4
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
       - name: Set up Python
-        uses: actions/setup-python@v5
+        uses: actions/setup-python@ea165f8d65b6e75b540449e92b4886f43607fa02
         with:
           cache: 'pip'
         with:
```

actionlint (1.7.12)

[error] key "with" is duplicated in element of "steps" section. previously defined at line:121,col:9

(syntax-check)